### DISCUSSION TITLE
logger with current timestamp

### DISCUSSION THREAD

**Discussion by reedgra** [gabime/spdlog#2465] [Q&A]
I am replacing the inhouse logger with spdlog.  Our logger has a timestamp of the app-start appended to the filename.  Now I can see there is a daily option but is there one for creating a new log with the current timestamp?  The format of the stamp if it is of use `"appname-%Y%m%d%H%M%S.log"`

I am aware that I may need to roll my own, is there any particular gotchas I need to be aware of?

**Comment by tt4g** [comment]
You can replace template argument `FileNameCalc` of `daily_file_sink`.

https://github.com/gabime/spdlog/blob/b75edfafca581e33be29ab51b9546b2e955c2853/include/spdlog/sinks/daily_file_sink.h#L114-L115

https://github.com/gabime/spdlog/blob/b75edfafca581e33be29ab51b9546b2e955c2853/include/spdlog/sinks/daily_file_sink.h#L25-L38
